src/components/HeaderAuthOptions/HeaderSignUp/HeaderSignUp.js

```diff
@@ -15,20 +31,80 @@ class HeaderSignUp extends Component {
 
     this.state = {
       email: '',
+      openEmal: false,
+      openPass: false,
+      openUser: false,
+      emailAvail: false,
+      emailValid: false,
+      emailVal: false,
       username: '',
+      usernameVal: false,
+      usernameLength: false,
+      usernameTaken: false,
       identifier: '',
+      submit: false,
       password: '',
-      passwordConfirmation: ''
+      passwordVal: false,
+      passwordLength: false,
+      passwordCapital: false,
+      passwordLower: false,
+      passwordSpecial: false,
+      passwordNumber: false,
+      passwordConfirmation: '',
+      passwordConfirmationVal: false,
+      property: document.documentElement.style.setProperty('--border-show', 'none')
```

Comment:
what is `document.documentElement.style.setProperty('--border-show', 'none')`?

Reply:
I meant to comment on this. I'm using a CSS variable to change the border from 'none' to 'show' upon submit. This line resets the border display to 'none' when first loading the signup page. Without it, if you left the signup page and came back the borders would still be showing. We probably won't need the CSS variable when we implement the newer wire frames, but I wanted to leave it in for now... just in case.

Reply:
If you want this code to execute on load, why don't we add this to `componentDidMount()`

Do we need to save `property` to state?

Reply:
I'm setting state on line 83
```this.setState({ property: document.documentElement.style.setProperty('--border-show', 'solid') })```
Do you think it would still be a better idea to use `componentDidMount()`?

I was planning on addressing this issue when tackling the new wireframes. I'm not 100% sure we'll need this functionality at all. Most likely, we can just remove it.